UnitTests/GitUITests/UserControls/BlameControlTests.cs

```diff
@@ -51,8 +51,9 @@ public void SetUp()
 
             _gitBlameLine = new GitBlameLine(blameCommit1, 1, 1, "line1");
 
-            _sut = new BlameControl().GetTestAccessor();
-            _sut.Blame = new GitBlame(new GitBlameLine[]
+            _blameControl = new BlameControl();
+            var testAccessor = _blameControl.GetTestAccessor();
+            testAccessor.Blame = new GitBlame(new GitBlameLine[]
```

Comment:
💡 You can just inline the `testAccessor` variable here.

Reply:
This one is weird: It compiles with MSVS2019. But Intellisense (or maybe Re#) report an error

![grafik](https://user-images.githubusercontent.com/36601201/71736844-67175600-2e52-11ea-83aa-a8d849335747.png)

And it does not compile with the MSVS2017 of AppVeyor.